app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(team: list) -> int:
+    return sum(user.get_rating() for user in team)
+
+
+def elves_concert(elves: list) -> list:
+    result1 = []
+    for user_elf in elves:
+        result1.append(Elf.play_elf_song(user_elf))
```

Comment:
The method `play_elf_song` should be called on the instance `user_elf`, not on the `Elf` class. Change `Elf.play_elf_song(user_elf)` to `user_elf.play_elf_song()`.